boards/arm/frdm_k32l2b3/doc/index.rst

```diff
@@ -0,0 +1,230 @@
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/frdm_k32l2b3/frdm_k32l2b3_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The K32L2B3 SoC has five pairs of pinmux/gpio controllers, and all are currently enabled
+(PORTA/GPIOA, PORTB/GPIOB, PORTC/GPIOC, PORTD/GPIOD, and PORTE/GPIOE) for the FRDM-K32L2B3 board.
+
++-------+-------------+---------------------------+
+| Name  | Function    | Usage                     |
++=======+=============+===========================+
+| PTE31 | GPIO        | Red LED                   |
++-------+-------------+---------------------------+
+| PTD5  | GPIO        | Green LED                 |
++-------+-------------+---------------------------+
+| PTA1  | UART0_RX    | UART Console              |
++-------+-------------+---------------------------+
+| PTA2  | UART0_TX    | UART Console              |
++-------+-------------+---------------------------+
+| PTE24 | I2C0_SCL    | I2C                       |
++-------+-------------+---------------------------+
+| PTE25 | I2C0_SDA    | I2C                       |
++-------+-------------+---------------------------+
+
+
+System Clock
+============
+
+The K32L2B3 SoC is configured to use the 8 MHz external oscillator on the board
+with the on-chip FLL to generate a 48 MHz system clock.
+
+Serial Port
+===========
+
+The K32L2B3 UART0 is used for the console.
+
+USB
+===
+
+The K32L2B3 SoC has a USB FS controller that supports device through its
+micro USB connector (USB K32L2B3).
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`.
+
+Early versions of this board have an outdated version of the OpenSDA bootloader
+and require an update. Please see the `DAPLink Bootloader Update`_ page for
+instructions to update from the CMSIS-DAP bootloader to the DAPLink bootloader.
+
+Option 1: :ref:`opensda-daplink-onboard-debug-probe` (Recommended)
+------------------------------------------------------------------
+
+Install the :ref:`pyocd-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`opensda-daplink-onboard-debug-probe` to program
+the `OpenSDA DAPLink FRDM-K32L2B3 Firmware`_.
+
+Option 2: :ref:`opensda-jlink-onboard-debug-probe`
+--------------------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`opensda-jlink-onboard-debug-probe` to program
+the `OpenSDA J-Link FRDM-K32L2B3 Firmware`_.
```

Comment:
I see you've got a "OpenSDA J-Link Generic Firmware" link defined at the bottom of the doc, so this should probably refer to that?